packages/node/src/lib/process-request.ts

```diff
@@ -185,6 +185,10 @@ export default function processRequest(
   // req.originalUrl is express specific, req.url is node.js
   const reqUrl = new URL(req.originalUrl || req.url, 'https://readme.io');
 
+  if (req.headers.authorization) {
+    req.headers.authorization = mask(req.headers.authorization);
```

Comment:
For basic or bearer auth do we only want to mask the data after `basic ` and `bearer `?

Reply:
Good question, i'm actually not certain whether we want to hash the whole value or not. I think it depends how these values are used in the dash/hubs and whether we're splitting on those values before searching for them or not. What do you think?

Reply:
probably easier to just encode the whole thing because if we don't somewhere down the line I'm almost positive that we'll forget we encoded it and then have a user confused why their snippet with the partially encoded header doesn't work.

Reply:
As long as are standard across our hashing, we should be able to match a key in the user's cookie correctly. We're already breaking replayability when a user doesn't have a key (via the share functionality for example).